blocks/library/image/block.js

```diff
@@ -253,6 +253,10 @@ class ImageBlock extends Component {
 										height: currentHeight + delta.height,
 									} );
 								} }
+								onResizeStart={ ( event ) => {
+									// Stop image resizing firing multi-select
+									event.stopPropagation();
```

Comment:
How do we know that this is a `mousedown` event? (And I don't just mean the specific implementation of ResizableBox, but also abstractly whether we should rely on it to be implemented this way from an `onResizeStart` callback)

Reply:
Yeah, I'm not sure what else to do here  Because multi-select listens to anything that propagates, it has to be specifically stopped when something is handling the mouse interaction. We could bind to mousedown here I suppose at a higher level, but that would stop multi-select starting  from within an image. We'd also they have to work out what is starting a resize and what isn't. I'm not sure the multi-select from anywhere unless it's specifically stopped is going to scale well. Popovers can help, but they aren't going to change that a block implementer is going to need to stop any event that has a particular interaction in their block (like image resize)